.ci/.matrix_framework.yml

```diff
@@ -42,6 +42,7 @@ FRAMEWORK:
   - mysqlclient-newest
   - aiohttp-newest
   - aiopg-newest
+  - asyncpg-0.28
```

Comment:
Can you please remove this line? This file is for a subset of test matrices that we want to run on PRs. We typically only run the `-newest` tests here.